src/webui/www/private/scripts/client.js

```diff
@@ -1256,6 +1291,12 @@ window.addEventListener("DOMContentLoaded", function() {
             },
             loadMethod: 'xhr',
             contentURL: 'views/search.html',
+            require: {
+                js: ['scripts/search.js'],
```

Comment:
I think they're functionally equivalent, so would just be a matter of personal preference.

Reply:
I have thought a little more about it.
* If `search.html` always require `search.js`, then it is better to include the js within it, rather than externally.
* `<script>` can have `defer` attribute which might be able to speed up page loading.

ps. I would like to investigate whether if I can migrate all `require.js` and `require.css` to html equivalents.

Reply:
In my quick testing, including a `<script>` in `search.html` does not result in fetching that script. I'm not sure why.

> `<script>` can have `defer` attribute which might be able to speed up page loading.

FWIW I think the `<script>` tag may actually be slightly slower (though I doubt it'll matter in practice). With `require`, the JS will be loaded asynchronously with `search.html`, whereas with the `<script>` tag the JS won't be loaded until after `search.html` has been loaded.